column/api/run.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from flask import Flask
```

Comment:
I agree python is not the best choice, but we were kind of stuck with it based on the fact ansible is using python. I would prefer Go myself :)